.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Codespace with PowerShell, Pester, Invoke-Build, and .NET 8",
+  "image": "mcr.microsoft.com/vscode/devcontainers/base:ubuntu",
+  "features": {
+    "ghcr.io/devcontainers/features/powershell:1": {},
+    "ghcr.io/devcontainers/features/dotnet:1": {
+      "version": "8.0"
+    }
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-vscode.powershell",
+        "pspester.pester-test"
+      ]
+    }
+  },
+  "postCreateCommand": "pwsh -Command 'Install-Module -Name InvokeBuild -Force -SkipPublisherCheck'"
```

Comment:
Tested myself, and this part is missing Pester, and I'm thinking the running of Invoke-Build as well. When the codespace was up you can't immediately run the HelloWorld.ps1 example as Pode hasn't been compiled.

I'm thinking, what if we add a `setup.ps1` script in the `/.devcontainer` folder with:
```powershell
Install-Module -Name InvokeBuild -Force -SkipPublisherCheck
Install-Module -Name Pester -Force -SkipPublisherCheck
Invoke-Build Build
```

The `postCreateCommand` could then be `pwsh -File <path>/.devcontainer/setup.ps1` :)

Reply:
I understood.
I'm running invoke-build Test on the terminal 

Let me see if I can automate the build process

Reply:
Tested in a fresh codespace as well, and I can now run the HelloWorld.ps1 example on start :)